### ISSUE TITLE
NullPointException error

### ISSUE THREAD

**Issue by hoverPaul** [TeamAmaze/AmazeFileManager#1312]
occurance:
```
AndroidRuntime: java.lang.NullPointerException: You cannot start a load on a not yet attached View or a Fragment where getActivity() returns null (which usually occurs when getActivity() is called before the Fragment is attached or after the Fragment is destroyed).
06-23 10:26:30.157  8082  8082 E AndroidRuntime:        at com.bumptech.glide.util.Preconditions.checkNotNull(Preconditions.java:31)
06-23 10:26:30.157  8082  8082 E AndroidRuntime:        at com.bumptech.glide.Glide.getRetriever(Glide.java:665)
06-23 10:26:30.157  8082  8082 E AndroidRuntime:        at com.bumptech.glide.Glide.with(Glide.java:747)
06-23 10:26:30.157  8082  8082 E AndroidRuntime:        at com.amaze.filemanager.GlideApp.with(GlideApp.java:120)
06-23 10:26:30.157  8082  8082 E AndroidRuntime:        at com.amaze.filemanager.adapters.RecyclerAdapter.setItems(RecyclerAdapter.java:363)
06-23 10:26:30.157  8082  8082 E AndroidRuntime:        at com.amaze.filemanager.adapters.RecyclerAdapter.setItems(RecyclerAdapter.java:321)
06-23 10:26:30.157  8082  8082 E AndroidRuntime:        at com.amaze.filemanager.asynchronous.handlers.FileHandler.handleMessage(FileHandler.java:71)
06-23 10:26:30.157  8082  8082 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:106)
06-23 10:26:30.157  8082  8082 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:164)
06-23 10:26:30.157  8082  8082 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6518)
06-23 10:26:30.157  8082  8082 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
06-23 10:26:30.157  8082  8082 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
06-23 10:26:30.157  8082  8082 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
06-23 10:26:30.200  3034  3178 I ActivityManager: Showing crash dialog for package com.amaze.filemanager u0
```
Any good way to fix it?

Thanks

**Comment by TranceLove** [comment]
Can you provide more information about the crash?

e.g.
 * version (current Google Play Store's release or `master`?)
 * the Android device you're using
 * what were you doing when the crash occurred (browsing files on internal storage/SD card? Opening zip file?)

Thanks in advance.

**Comment by VishalNehra** [comment]
Should've been closed with #1352
